examples/bidder/http_bidder_test.cpp

```diff
@@ -95,49 +103,63 @@ int main(int argc, char *argv[]) {
         LOG(error) << e.what();
         return 0;
     }
-    exchange_handler<DSL::GenericDSL<>> bid_handler(std::chrono::milliseconds(config.data().timeout));
+    
+
+    using bid_handler_type = exchange_handler<DSL::GenericDSL<>, vanilla::VanillaRequest>;
```

Comment:
I see now what's going on now ... the exchange_handler<DSL> was designed to handle request in the generic way relying on DSL::serialized_type , DSL::deserialized_type , if we move it to vanilla::VanillaRequest then we'll need to maintain conversions for all different DSL types inside of 
core/bid_request.hpp .
It basically breaks our model ?
I also see that user matching relies on vanilla::VanillaRequest as it has user_info.
We need to think about it.